tests/test_mount_c_api.cpp

```diff
@@ -0,0 +1,71 @@
+#include <glib.h>
+#include <locale.h>
+#include "lensfun.h"
+
+#include <array>
+
+void test_mount_c_api()
+{
+    lfMount* mount = lf_mount_create();
+    g_assert_nonnull(mount);
+    g_assert_false(lf_mount_check(mount));
+
+    mount->Name = lf_mlstr_add(mount->Name, "en", "Nikon F");
+    g_assert_true(lf_mount_check(mount));
+    g_assert_null(lf_mount_get_compats(mount));
+    // g_assert_null(mount->Compat); // deprecated
```

Comment:
remove this.

Reply:
and the other commented out  // deprecated

Reply:
done